drivers/saul/init_devs/auto_init_l3gxxxx.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2018 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/*
+ * @ingroup     sys_auto_init_saul
+ * @brief       Auto initialization of L3GD20H 3-axis gyroscope sensor
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ * @file
+ */
+
+#ifdef MODULE_L3GD20H
```

Comment:
```suggestion
#ifdef MODULE_L3GXXXX
```
same for the rest of the file